src/NightwatchInitiator.ts

```diff
@@ -568,14 +566,13 @@ export default class NightwatchInitiator {
   }
 
   async installWebdrivers(webdriversToInstall: string[]) {
```

Comment:
I'm not sure if this function is appropriate for those who prefer end-to-end or component testing since installing Webdriver is managed by selenium. In my opinion, this function should only be used in mobile testing setups because we need to configure the chromedriver/geckodriver based on the devices in the emulator.

Reply:
This needs fixing, working on this.

Reply:
I'll take care of this. I'm currently working on fixing `create-nightwatch` to work with Appium v2.

Reply:
But I don't understand what you mean by this:
>  In my opinion, this function should only be used in mobile testing setups

This function is currently being called in case of mobile app testing setup only, because that's when we'll get something in `webdriversToInstall`. That and if Safari browser is selected, because safaridriver is not handled by selenium-manager, and need to be enabled manually.

Reply:
I think @garg3133 is correct here. It is also something that we can refactor later